Project-Lily/Project-Lily/DogListViewController.swift

```diff
@@ -108,7 +109,7 @@ extension DogListViewController:  UITableViewDataSource {
             dogs = filteredDogArray[indexPath.row]
         } else {
             
-            dogs = dogDatabase.dogsArray[indexPath.row]
+            dogs = dogListVM.dogs[indexPath.row]
```

Comment:
Whitespace handling is different in these 2 `else` blocks. I suspect most linters (like the SwiftLint we just implemented on our project!) will prompt you with a warning for whitespace like you've got on line `111`.  Lines `164`--`167` are looking good. I'd follow that style.